cmd/geth/main.go

```diff
@@ -400,6 +401,9 @@ func run(ctx *cli.Context) {
 
 	ethereum, err := eth.New(cfg)
 	if err != nil {
+		if datadirInUseErrRE.MatchString(err.Error()) {
+			err = fmt.Errorf("%v (another instance of geth is already running with the same data directory '%v')", err, ctx.GlobalString(utils.DataDirFlag.Name))
+		}
```

Comment:
can this be a platform thing? On Mac I get:

     Fatal: blockchain db err: resource temporarily unavailable (another instance of geth is already running with the same data directory 'tmp/try2')